onadata/apps/api/tests/viewsets/test_xform_viewset.py

```diff
@@ -1389,7 +1389,7 @@ def test_partial_update(self):
             view = XFormViewSet.as_view({
                 'patch': 'partial_update'
             })
-            title = u'مرحب'
```

Comment:
I think there is plenty of places where Unicode characters are tested. I don't think it's worth having here.